src/Lean/PremiseSelection/MePo.lean

```diff
@@ -43,8 +43,9 @@ def frequencyScore (frequency : Name → Nat) (relevant candidate : NameSet) : F
 
 def unweightedScore (relevant candidate : NameSet) : Float := weightedScore (fun _ => 1) relevant candidate
 
--- The values of p := 0.6 and c := 2.4 are taken from the MePo paper, and need to be tuned.
-def mepo (initialRelevant : NameSet) (score : NameSet → NameSet → Float) (accept : ConstantInfo → CoreM Bool) (p : Float := 0.6) (c : Float := 2.4) : CoreM (Array (Name × Float)) := do
+-- The value c = 0.5 essentially forces the `while` loop to only iterate once, which preliminary
+-- experiments found to be optimal.
```

Comment:
This seems really implausible to me, given the original MePo paper came up with 2.4. I'd want some pretty compelling evidence to back up this claim.

Reply:
Let's back out this change for now until we can discuss benchmarking evidence?